geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/v1/utilities/ProtobufResponseUtilities.java

```diff
@@ -61,4 +63,15 @@
   public static ClientProtocol.ErrorResponse makeErrorResponse(ConnectionStateException exception) {
     return makeErrorResponse(exception.getErrorCode(), exception.getMessage());
   }
+
+  public static ClientProtocol.ErrorResponse makeErrorResponse(Exception exception) {
+    if (exception instanceof ConnectionStateException) {
+      return makeErrorResponse((ConnectionStateException) exception);
+    }
+    return ClientProtocol.ErrorResponse.newBuilder()
+        .setError(BasicTypes.Error.newBuilder().setErrorCode(BasicTypes.ErrorCode.SERVER_ERROR)
+            .setMessage(exception.getClass().getSimpleName() + exception.getMessage()))
```

Comment:
Should there be some separator text (e.g., `": "`) between the simple name and the message?